nmostesting/suites/IS0401Test.py

```diff
@@ -1413,6 +1413,46 @@ def test_24_01(self, test):
 
         return test.UNCLEAR("No Source or Flow resources were found on the Node")
 
+    def test_25(self, test):
+        """Receivers expose expected 'caps' for their API version"""
+
+        api = self.apis[NODE_API_KEY]
+
+        if self.is04_utils.compare_api_version(api["version"], "v1.1") < 0:
+            return test.NA("Capabilities are not used before API v1.1")
+
+        receivers_valid, receivers_response = self.do_request("GET", self.node_url + "receivers")
+
+        no_receivers = True
+        if receivers_valid and receivers_response.status_code == 200:
+            try:
+                for receiver in receivers_response.json():
+                    no_receivers = False
+                    if "media_types" not in receiver["caps"]:
+                        return test.WARNING("Receiver 'caps' should include a list of accepted 'media_types', unless "
+                                            "this Receiver can handle any 'media_type'",
+                                            "https://amwa-tv.github.io/nmos-discovery-registration/tags/v1.3/docs/"

```

Comment:
In other places in the test suite, we've used links with `api["spec_branch"]`. I don't know if that's a good idea here since we didn't merge the documentation of `caps` back to the v1.2.x branch...